process/project_milestones.md

```diff
@@ -0,0 +1,64 @@
+# Cloud Native Milestones
+
+Milestones are intended to outline checkpoints for projects, that have assisted past projects as they grew and mature in the cloud native ecosystem. They are NOT requirements for moving between levels, please refer to [the graduation criteria](https://github.com/cncf/toc/blob/main/process/graduation_criteria.md) for requirements. 
+
+When leveraged, projects may find their path towards Graduation is more structured, clear, achievable, and allows them to reach maturity in a more robust and well understood manner. Milestones do not guarantee a project’s graduation within the CNCF, rather, they are a collection of key activities that have helped other projects achieve maturity, stability, and adoption. Even after projects are graduated, they may leverage milestones as a mechanism to continue sustaining their project in a manner expected by their community and adopters particularly as they experience change and turnover. The community is welcome to submit PRs to improve and add additional milestones based on their experiences in maturing their projects.
+
+## Sandbox
+
+Some milestones may not apply to all projects and should be leveraged as guiding points. Milestones are described as the desired state with examples for how to achieve this. They are not requirements.
+* Project has the basics of security underway
+  * Completion of the [self-assessment](https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md)
+* Project holds community meetings or active discussions
+  * Zoom with meeting notes occurring more than twice a quarter (or community.cncf.io!) 
+  * Use of GitHub discussions, issues, or a group messaging service (slack, wechat, etc.)
+* Project guides community contributions
+  * PRs from community members receive timely feedback and discussion from the maintainers
+* Project works with an angel adopter
+  * Partners closely with the project to address organizational needs of the project early on
+* Project has a second organization contributing
+  * May begin path to maintainership
+* Project begins solidifying a versioning schema and release cadence
+  * A roadmap or other document clearly defines project direct or initiatives that may align with releases
+  * Releases begin to establish a pattern of regularity
+* Project has engaged a [Technical Advisory Group](/tags/README.md) for feedback on some portion of the project relevant to the domain
+* Project has a clearly discoverable governance doc that covers the basics of decision making and how to earn permissions to approve pull request, sometimes referred to as "the commit bit"
+* Project has an angel adopter that assists in stability and production runtime or deployment
+* Project has 3 angel adopters
+  * May be from different industry verticals
```

Comment:
Not quite understanding the language around "angel adopters", can this explain better?